python/llm/example/CPU/PyTorch-Models/Model/Qwen-VL-Chat/chat.py

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from transformers import AutoModelForCausalLM, AutoTokenizer
+from transformers.generation import GenerationConfig
+import torch
+import time
+import os
+import argparse
+from bigdl.llm import optimize_model
+torch.manual_seed(1234)
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Predict Tokens using `chat()` API for Qw-VL-Chat model')
+    parser.add_argument('--repo-id-or-model-path', type=str, default="Qwen/Qwen_VL_Chat",
+                        help='The huggingface repo id for the Qwen_VL_Chat model to be downloaded'
+                             ', or the path to the huggingface checkpoint folder')
+    parser.add_argument('--n-predict', type=int, default=32, help='Max tokens to predict')
+    
+    current_path = os.path.dirname(os.path.abspath(__file__))
+    args = parser.parse_args()
+    model_path = args.repo_id_or_model_path
+
+    # Load model
+    model = AutoModelForCausalLM.from_pretrained(model_path, device_map="cpu",  trust_remote_code=True).eval()
+
+    # With only one line to enable BigDL-LLM optimization on model
+    # Note that currently there is no optimization support 'c_fc' and 'out_proj' moudule, 'modules_to_not_convert' should be set
```

Comment:
We could change the comment to 

For successful BigDL-LLM optimization on Qwen-VL-Chat, skip the 'c_fc' and 'out_proj' modules during optimization.